src/util/cloud_utils.js

```diff
@@ -22,7 +22,7 @@ function find_cloud_connection(account, conn_name) {
 
 
 function get_signed_url(params) {
-    let agent = http_utils.get_unsecured_http_agent(params.endpoint);
+    const agent = http_utils.get_http_agent(params.endpoint, { rejectUnauthorized: false });
```

Comment:
Why did you cancel the secure vs unsecure separation?
It was intentional to have a separate identifier for unsecure calls.

Reply:
In the old code, the only case for a custom agent was an unsecured agent. Now we need an agent also because of the proxy stuff. So I just generalized the agent creation code and passed the un-secure flag when necessary  (as part of  an options object)

Reply:
I still prefer that it's a separate function.

Reply:
This is not a problem I can add a specialized function just got that

Reply:
great, 2 specialized functions

Reply:
make the internal function actually internal

Reply:
It is now internal